Libraries/Modal/Modal.js

```diff
@@ -159,19 +158,7 @@ class Modal extends React.Component<Props> {
     this._identifier = uniqueModalIdentifier++;
   }
 
-  static childContextTypes:
-    | any
-    | {|virtualizedList: React$PropType$Primitive<any>|} = {
-    virtualizedList: PropTypes.object,
```

Comment:
I think this is resetting the usage of the context that is created in virtualizedlist, so pulling this into a separate context won't work because they'll be different values. I think this needs to reset whatever provider is created from VirtualizedList